docs/man/reuse-lint.rst

```diff
@@ -103,3 +103,12 @@ Options
 .. option:: -h, --help
 
   Display help and exit.
+
+Environment
+-----------
+
+.. envvar:: REUSE_OUTPUT_FORMAT
+
+   Specifies output format, one of ``plain``, ``lines``, ``github``, ``json``
+
+   It behaves same as corresponding command line options.
```

Comment:
There should be a documented order of precedence here. If both `REUSE_OUTPUT_FORMAT` and a CLI flag are used, which takes precedence?

As written in the code, I believe the env var takes precedence. I do not know if this is desirable behaviour.

Reply:
Additionally, should `quiet` be a valid value here?

Reply:
Additionally, what should happen if the value of `REUSE_OUTPUT_FORMAT` is invalid?

Reply:
Ah, seems like I didn't notice this feedback, sorry.

With the current implementation, the environment variable takes precedence. This is intentional to allow having command line configuration and override it by environment if executed in environment which might want different output (as GitHub).

Unknown values are simply discarded, but it might be a better approach to issue a warning.